odoo_sh/getting_started/create.rst

```diff
@@ -0,0 +1,148 @@
+* to create a webhook to be notified each time you push changes,
+* to commit changes to make your deployment easier, merging branches or adding new `submodules <https://git-scm.com/book/en/v2/Git-Tools-Submodules>`_ for example.
+
+Submit your project
+===================
+
+Choose if you want to start from scratch by creating a new repository, or if you want to use an existing repository.
+
+Then, choose a name or select the repository you want to use.
+
+Choose the Odoo version you want to use. If you plan to import an existing database or an existing set of applications, you might need to choose the according version. If you start from scratch, use the latest version.
+
+Enter your *subscription code*. This is also called *subscription referral* or *contract number*.
+
+For partners, Odoo.sh is free. If the free offer changes in the future, we guarantee that any project created under this offer will remain free for the same set of features.
+
+For customers, your Enterprise subscription needs to include Odoo.sh.
+
+After submitting the form, if you are notified your subscription is not valid, it either means:
+
+* it is not an existing subscription,
+* it is not a partnership subscription,
+* it is an enterprise subscription, but which does not include Odoo.sh,
+* it is neither a partnership subscription or an enterprise subscription (e.g. an online subscription).
+
+.. image:: ./media/deploy-form.png
+   :align: center
+
+You're done !
+=============
+
+You can begin to use Odoo.sh. Your first build is about to be created. You will soon be able to connect to your first database.
+
+.. image:: ./media/deploy-done.png
+   :align: center
+
+Import your database
+====================
+
+You can import your database in your Odoo.sh project as long as this is an Odoo 10.0 or 11.0 database.
+
+Push your modules in production
+-------------------------------
+
+If you use community or custom modules, add them in a branch in your Github repository.
+Databases hosted on the Odoo.com online platform do not have any custom modules.
+Users of these databases can therefore skip this step.
+
+You can structure your modules as you wish, Odoo.sh will automatically detect the folders containing Odoo addons.
+For instance, you can put all your modules folder in the root directory of your repository,
+or group the modules in folders by categories that you define (accounting, project, ...).
+
+For community modules availabe in public Git repositories,
+you can also consider to add them using :ref:`Submodules <odoosh-advanced-submodules>`.
+
+Then, either :ref:`make this branch the production branch <odoosh-gettingstarted-branches-stages>`,
+or :ref:`merge it into your production branch <odoosh-gettingstarted-branches-mergingbranches>`.
+
+Download a backup
+-----------------
+
+On-premise databases
+~~~~~~~~~~~~~~~~~~~~
+
+Access the URL :file:`/web/database/manager` of your on-premise database and download a backup.
```

Comment:
If you cannot access the database manager, it may have been disabled by your system administrator. ( see https://www.odoo.com/documentation/11.0/setup/deploy.html#database-manager-security )